src/imitation/scripts/config/train_preference_comparisons.py

```diff
@@ -38,6 +38,8 @@ def train_defaults():
     normalize = True  # Use VecNormalize
     normalize_kwargs = {"norm_reward": False}  # kwargs for `VecNormalize`
 
+    checkpoint_interval = 0  # Num epochs between checkpoints (<0 disables)
```

Comment:
`=0` disables all but final one right (which I'm not sure I'd call a "checkpoint" anyway)?

```suggestion
    checkpoint_interval = 0  # Num epochs between checkpoints (<0 disables, =0 final only)
```